ods_ci/tasks/Resources/Provisioning/GPU/provision-gpu.sh

```diff
@@ -4,9 +4,10 @@ set -e
 # Optional params
 INSTANCE_TYPE=${1:-"g4dn.xlarge"}
 PROVIDER=${2:-"AWS"}
+GPU_COUNT=${3:-"1"}
 KUSTOMIZE_PATH="$PWD/tasks/Resources/Provisioning/Hive/GPU"
 MACHINESET_PATH="$KUSTOMIZE_PATH/base/source-machineset.yaml"
-
+PROVIDER_OVERLAY_DIR=$PROVIDER
```

Comment:
maybe you can change it to something like:
```suggestion
PROVIDER_OVERLAY_DIR=$KUSTOMIZE_PATH/overlays/$PROVIDER
```
and it will be more readable in the lines

Reply:
thanks for the suggestion. I've update the code upon it. Pls review